pkg/apis/cartographer/v1alpha1/workload.go

```diff
@@ -82,9 +82,14 @@ type WorkloadBuild struct {
 }
 
 type WorkloadStatus struct {
-	ObservedGeneration int64              `json:"observedGeneration,omitempty"`
```

Comment:
After removing these - will the apps plugin work with an older version of Cart Workload API? Is this change backward compatible?

Reply:
it was [moved](https://github.com/vmware-tanzu/apps-cli-plugin/pull/66/files#diff-7d0ecc1b67d2a8c12986aba3606086e9a310664b69e606aa11bccfb1170e1f35R23-R32), not removed